features/news.feature

```diff
@@ -1,35 +1,33 @@
 @news
 Feature: Different options for news. I would like to test whether users with different permissions can see my news
 
-	Background: I am logged in as a teacher and I create news
+	Background:
 		Given user arrives on the Schul-Cloud homepage
 
-    @createInstantNews
-    Scenario Outline: User can see the news
-        Given teacher logs in with email '<teacherEmail>' and password '<teacherPassword>'
-        And teacher performs first login actions: data protection acceptance
-        When teacher creates news with title '<newsTitle>', content '<newsContent>' and current date
-        And teacher logs out
-        And student logs in with email '<studentUsername>' and password '<studentPassword>'
-        And student performs first login actions: data protection acceptance, password change '<newStudentPassword>'
-        And clicks left navigation item 'news'
-        Then teacher should see that news with title '<newsTitle>' is visible on the list
+	@createInstantNews
+	Scenario Outline: As a user, I want to be able to see the news
+		When <userRole> logs in with email '<teacherEmail>' and password '<teacherPassword>'
+		And <userRole> performs first login actions: data protection acceptance
+		When <userRole> creates news with title '<newsTitle>', content '<newsContent>' and current date
+		And <userRole> logs out
+		And student logs in with email '<studentUsername>' and password '<studentPassword>'
```

Comment:
why are we not parameterize the student role?

Reply:
Maybe because this is within the same test run and would possibly return an error.